googl.py

```diff
@@ -28,8 +28,7 @@
 __version__ = "0.1.1"
 __author__ = "Ivan Grishaev"
 
-import urllib
-import urllib2
+import urllib.request, urllib.error, urllib.parse
```

Comment:
some notes

- that will break py2.x compatibility
- use one import per line (pep8)
- I believe requests package should be used.